consensus/snapshot/BUILD

```diff
@@ -1,15 +1,37 @@
+filegroup(
+    name = "snapshot_files",
```

Comment:
We should have a `config_setting` together with `select(`'s here to pick the right snapshot file group :thinking:

Reply:
As discussed, this will be handled in another PR where the network type would be a compile-time option.